### PR TITLE
Remove binary log from mysql config.

### PR DESCRIPTION
**Pull request by fdurand**
# Description
mysql binary log fill the disk and it's useless.

# Impacts
Mysql

# Delete branch after merge
YES

# NEWS file entries

## Enhancements
* Removed binary log from mysql configuration

### CODE REVIEW

File: conf/mariadb/mariadb.conf.tt.example [hunk 1]
```diff
@@ -46,7 +46,7 @@ thread_concurrency = [% thread_concurrency %]
 [% IF cluster_enabled %]
 # Replication Master Server (default)
 # binary logging is required for replication
```
Comment (julsemaan) [inline conf/mariadb/mariadb.conf.tt.example]: lets remove that comment or adjust it

**Comment by louismunro** [comment]
LGTM. Merging.

**Comment by julsemaan** [comment]
Sounds like we'll need a best practice check script for that

**Comment by extrafu** [comment]
👍 for the script

**Pull request merged**